omero/developers/Modules/Search.rst

```diff
@@ -39,7 +39,16 @@ Queries
 Search queries are very similar to Google searches. When search terms
 are entered without a prefix ("name:"), then the default field will be
 used which combines all available fields. Otherwise, a prefix can be
-added to restrict the search.
+added to restrict the search. The search terms will be parsed as if there was an "OR" operator
+between them (see examples in the "Indexing" paragraph).
+The tokenizing happens on all non-alpha-numerical characters, such as underscore,
+hyphen etc. Again, when the term is tokenized, 
+the query is built by combining the tokens with an "OR".
+The search terms or the tokens created from them as above 
+must precisely match the indexed entries. 
+This means for example that a search term `tes` 
+will **not** match the indexed entry `test` and the search
+will accordingly give no result. 
```

Comment:
Does the following sound OK (trying to separate the tokenization and "OR")?

The search term is first split into "tokens" and these are combined into a search query. The tokenizing happens on all non-alpha-numerical characters, such as underscore, hyphen etc.  The query is built by combining the tokens with an "OR" operator (see examples in the "Indexing" paragraph). The search terms or the tokens created....

Reply:
Yes, I am adding the word "space", as now your formulation encompasses also the most common case of what is perceived by user as  "multiple terms entry" such as ``GFP H2B kidney``